packages/core/expressions/src/utils.ts

```diff
@@ -59,4 +59,14 @@ export const transformCheckRequest = (request: Partial<Request>): string | undef
       return i18n.t('errors.methodShouldCapitalized')
     }
   }
+
+  if (request.headers) {
+    Object.keys(request.headers).forEach(key => {
+      const values = request.headers![key]
+      // transform header values to array
+      if (typeof values === 'string' && values.indexOf(',') !== -1) {
+        request.headers![key] = values.split(',').map(value => value.trim())
```

Comment:
Should we filter out empty values from the header's value set? 

e.g. Now: `a,b,c,` → `["a", "b", "c", ""]`

<img width="325" alt="Screenshot 2024-08-15 at 11 27 09" src="https://github.com/user-attachments/assets/4acd268c-59f8-4e3f-bc66-e7af1433717b">

Reply:
I don’t think so. Making it strict would require handling too many cases, which isn’t necessary and meaningful.